client/shared/src/components/FileMatch.tsx

```diff
@@ -128,10 +124,7 @@ export const FileMatch: React.FunctionComponent<Props> = props => {
                         target="_blank"
                         rel="noopener noreferrer"
                         data-tooltip={badge.hoverMessage}
-                        className={classNames(
-                            'badge badge-secondary text-muted text-uppercase file-match__badge',
-                            isRedesignEnabled && 'badge-sm'
-                        )}
+                        className="badge badge-secondary text-muted text-uppercase file-match__badge badge-sm"
```

Comment:
```suggestion
                        className="badge badge-secondary badge-sm text-muted text-uppercase file-match__badge"
```

Nitpick on class ordering :)